docs/content/guides/recipes/Terraform/howto-private-registry/snippets/env.bicep

```
@@ -0,0 +1,52 @@
+import radius as radius
+
+//ENV
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+  name: 'prod'
+  properties: {
+    compute: {
+      kind: 'kubernetes'
+      resourceId: 'self'
+      namespace: 'default'
+    }
+    recipeConfig: {
+      terraform: {
+        authentication:{
+          git:{  
+            'my-access-token':{
+              'github.com':{
```

```suggestion
              'dev.azure.com':{
```
lets change it to match the template path